client/ayon_core/hosts/maya/plugins/publish/validate_instance_in_context.py

```diff
@@ -34,6 +34,7 @@ class ValidateInstanceInContext(pyblish.api.InstancePlugin,
     ]
 
     def process(self, instance):
+        return
```

Comment:
```suggestion
        return
```
Maybe it's better idea to skip the validator for just layout family? Right now the validator would be skipping for any families for this adjustment.

Reply:
so like this. Just add the families field below the Validate instance in context field?
```
    ValidateInstanceInContext: BasicValidateModel = SettingsField(
        default_factory=BasicValidateModel,
        title="Validate Instance In Context",
        section="Validators")
    families: list[str] = SettingsField(
        default_factory=list, title="Families")
```

> (and create proper default values)

How do I create default values?

Reply:
> so like this. Just add the families field below the Validate instance in context field?
> 
> ```
>     ValidateInstanceInContext: BasicValidateModel = SettingsField(
>         default_factory=BasicValidateModel,
>         title="Validate Instance In Context",
>         section="Validators")
>     families: list[str] = SettingsField(
>         default_factory=list, title="Families")
> ```
> 
> > (and create proper default values)
> 
> How do I create default values?

Consider this pesudo code:
```

class ValidateInstanceInContextModel(BaseSettingsModel):
    enabled: bool = SettingsField(title="Enabled")
    optional: bool = SettingsField(title="Optional")
    active: bool = SettingsField(title="Active")
    families: list[str] = SettingsField(
        default_factory=list, title="Families")


class PublishersModel(BaseSettingsModel):
    ValidateInstanceInContext: BasicValidateModel = SettingsField(
        default_factory=ValidateInstanceInContextModel,
        title="Validate Instance In Context",
        section="Validators"
    )


DEFAULT_PUBLISH_SETTINGS = {
    "ValidateInstanceInContext": {
        "enabled": True,
        "optional": True,
        "active": True,
        "families": ["layout"]
    }
}
```

Reply:
You may need to change `families ` in AYON settings to `excluded_families` so that the system won't get confused on `families` and the families to be excluded.  Consider this pesudo code.
```
from maya import cmds
class ValidateInstanceInContext(pyblish.api.InstancePlugin,
                                OptionalPyblishPluginMixin):
    """Validator to check if instance asset match context asset.
    When working in per-shot style you always publish data in context of
    current asset (shot). This validator checks if this is so. It is optional
    so it can be disabled when needed.
    Action on this validator will select invalid instances in Outliner.
    """
    order = ValidateContentsOrder
    label = "Instance in same Context"
    optional = True
    hosts = ["maya"]
    actions = [
        ayon_core.hosts.maya.api.action.SelectInvalidAction, RepairAction
    ]
    excluded_families = []

    def process(self, instance):
        families = instance.data["families"]
        excluded_product_type = families.intersection(self.excluded_families)
        if excluded_product_type:
            self.log.debug("Skipping Validate Instance In Context")
        
        # Some codes for checking on current context
```

Reply:
Validate Instance in Context has no excluded families attribute.
![image](https://github.com/ynput/ayon-core/assets/10794257/6686b647-f1e8-4bd7-8142-c8635a931fbd)

Should I just hardcode the excluded families on the class to include layout.

```
class ValidateInstanceInContext(pyblish.api.InstancePlugin, OptionalPyblishPluginMixin):
    ...
    excluded_families = ["layout"]
```

Reply:
Have you built the sever addons and install maya addon into the AYON server? It seems that your maya addon is not at the latest version with this branch as I didn't notice the `families` as being part of the AYON setting as your screengrab shown. 
Speaking of the `excluded_families`, You can just add `excluded_families` into AYON settings and the `excluded_families` would be registered by this case.

Reply:
> Have you built the sever addons and install maya addon into the AYON server? 
How do I do this?

By the way, I updated my Maya adds bundle, and I still don't see a "families" attribute.
![image](https://github.com/ynput/ayon-core/assets/10794257/92212744-03b4-4820-87fe-1c25746cf1f1)

Reply:
i have commented the code below.
After the adjustment of the code, it should be like this.
![image](https://github.com/ynput/ayon-core/assets/64118225/778a3bc7-0a9d-484b-9f8d-65aa3370d5fb)